### ISSUE TITLE
Causing aroung 800ms delay between version 3.2.0 and 3.3.0

### ISSUE THREAD

**Issue by Deepanshu2017** [sindresorhus/windows-release#16]
The scenario -
- I have an Azure NodeJS function
  - Azure function has a dependency on CosmosDB NodeJS package
  - [Azure Cosmos DB NodeJS package](https://github.com/Azure/azure-sdk-for-js/blob/2e3cf69ad5363e9c36eb02e7035b5176515d1698/sdk/cosmosdb/cosmos/package.json#L89) has dependency over [os-name package](https://github.com/sindresorhus/os-name#readme)
  - os-name package has the dependency over windows-release package

I had a production system running on Cosmos NodeJS package (version 3.6.2) and recently I made some changes and tried to deploy the new code over the production. I started to see the latency of around 800ms in cosmos DB insertion call. The cosmos DB package version was the same so the issue was in one of the dependency. I manually checked each and every package on which Cosmos DB and it's dependencies are dependent on (trust I checked a lot of code and releases) and could drill down to the specific version which is causing the issue.

The Windows-release package version 3.3.0 is causing the Cosmos to delay the result by 800ms to 1000ms. While with Windows-release package version 3.2.0 Cosmos DB doesn't show this much of delay. (Given that everything in the system was constant and every other factor which might impact the results had been eliminated).

When comparing Windows-release 3.3.0 with Windows-release 3.2.0, I found that the only part in the code that has been changed is the below snippet

https://github.com/sindresorhus/windows-release/blob/39b78e784527b9202968bf518640283f57541fe1/index.js#L36-L40

Basically Windows-release moved from `wmic` to use `PowerShell` by default [in this commit](https://github.com/sindresorhus/windows-release/commit/75916af29c166a9c2cd778b61fe52738614b8259) and in case `PowerShell` fails, fallback to `wmic` and I suspect that due to this fallback and changing default from `wmic` to `PowerShell` is causing the delay.

**Comment by Deepanshu2017** [comment]
SideNote:
I don't necessarily know whether I should create this issue here or in the os-name package or in CosmosDB. But anyway creating the issue here.
Also seems like, you are the owner of this package as well as the owner of os-name package.

**Comment by sindresorhus** [comment]
// @landsfiskalen

**Comment by landsfiskalen** [comment]
Hmmm, weird. @Deepanshu2017 could you try and change the order of the powershell and wmic command?

**Comment by Deepanshu2017** [comment]
@landsfiskalen But the thing is, I really don't know if I can do that in your code and push the deployment since it was on the Azure function app.

The confirmed thing is, changing from 3.3.0 to 3.2.0 made resolved everything.

**Comment by Deepanshu2017** [comment]
One thing I can try is, try to run the 3.3.0 and 3.2.0 on my local system and check if I can replicate the issue on local.

**Comment by landsfiskalen** [comment]
That would be great if you could!
